### PR TITLE
Commands in readme

### PR DESCRIPTION
**Pull request by orhovy**
Added a validation regarding if all commands exist in the readme file. 

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/36716
waiting for: https://github.com/demisto/etc/issues/38462

## Description
Added a validation regarding if all commands exist in the readme file. 

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
Maybe add the mirroring commands to the ignored list too. what do you think? (get-remote-data, get-modified-remote-data, update-remote-system)

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 1]
```diff
@@ -1101,6 +1101,45 @@ def is_mapping_fields_command_exist(self) -> bool:
...
         return True
 
+    def are_commands_in_readme(self) -> bool:
+        """
+        Check if commands from the YML file exist in the README file.
+        Returns:
+            True if all commands exist, else False.
+        """
+        valid = True
+        commands_to_ignore = ['get-mapping-fields']
```
Comment (bakatzir) [inline demisto_sdk/commands/common/hook_validations/integration.py:1111]:
```suggestion
        commands_to_ignore = ['get-mapping-fields', 'get_remote_data_command', 'update-remote-system', 'get-remote-data'] # fetch-incidents, test-module commands are not defined in the yml, hence not needed to be ignored.
```

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 2]
```diff
@@ -1101,6 +1101,45 @@ def is_mapping_fields_command_exist(self) -> bool:
...
+        dir_path = os.path.dirname(self.file_path)
+
+        if not os.path.exists(os.path.join(dir_path, 'README.md')):
+            return True
```
Comment (bakatzir) [inline demisto_sdk/commands/common/hook_validations/integration.py:1115]: how many missing READMEs do we have? should we add a warning for a missing readme? something like that this validation will not run if...

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 3]
```diff
@@ -1101,6 +1101,45 @@ def is_mapping_fields_command_exist(self) -> bool:
...
+        # get README file's content
+        with open(readme_path, 'r') as readme:
+            readme_content = readme.read()
+
+        commands = self.current_file.get("script", {})
+        commands_not_in_yml = []
+
+        # handles scripts
+        if not commands:
+            return True
```
Comment (bakatzir) [inline demisto_sdk/commands/common/hook_validations/integration.py:1128]: why is this needed? If I understood correctly, scripts will not get to `integration.py`.

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 4]
```diff
@@ -1101,6 +1101,45 @@ def is_mapping_fields_command_exist(self) -> bool:
                     return False
         return True
 
+    def are_commands_in_readme(self) -> bool:
```
Comment (bakatzir) [inline demisto_sdk/commands/common/hook_validations/integration.py:1104]: don't we need to add `are_commands_in_readme` to `is_valid_file`?

File: demisto_sdk/commands/common/hook_validations/integration.py (hunk 2 above)
Comment (orhovy) [reply demisto_sdk/commands/common/hook_validations/integration.py:1115]: I think that it is not relevant to this validation, If Im not mistaken we do have a validation that makes sure that a readme exists, if not , need to open a different issue regarding it.

File: demisto_sdk/commands/common/hook_validations/integration.py (hunk 4 above)
Comment (orhovy) [reply demisto_sdk/commands/common/hook_validations/integration.py:1105]: yes, I did, Not sure why it wasnt pushed. Now it should be.

File: demisto_sdk/commands/common/hook_validations/integration.py (hunk 3 above)
Comment (orhovy) [reply demisto_sdk/commands/common/hook_validations/integration.py:1129]: You are right, removed.

**Comment by bakatzir** [comment]
aside from the run on the content repo, Please add to the CHAGNELOG as well.
@ilappe FYI for the addition of this new validation as the one who lately released the sdk.

**Review by bakatzir — APPROVED** [review]
